cmd/sandbox-ce/cmd/web.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+	"github.com/smacker/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type webCmd struct {
+	Command `name:"web" short-description:"Web"`
+}
+
+func (c *webCmd) Execute(args []string) error {
+	if err := <-OpenUI(10 * time.Second); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func init() {
+	rootCmd.AddCommand(&webCmd{})
+}
+
+func openUI() error {
+	var stdout, stderr bytes.Buffer
+	for {
+		if err := compose.RunWithIO(context.Background(),
+			os.Stdin, &stdout, &stderr, "port", "superset", "8088"); err == nil {
+			break
+		}
+
+		time.Sleep(1 * time.Second)
+	}
+
+	address := strings.TrimSpace(stdout.String())
```

Comment:
For me this is opening `http://0.0.0.0:8088`, which feels very wrong as it is not a valid destination IP, and I'm still surprised every time I realize chrome treats it like localhost. Maybe we could replace `0.0.0.0` with `127.0.0.1`?